### PR TITLE
ENH: Add hippocampus segmentation config file

### PR DESCRIPTION
**Pull request by annaschroder**
Adding a config file for hippocampal segmentation

### CODE REVIEW

File: InnerEye/ML/configs/segmentation/Hippocampus.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+from InnerEye.ML.deep_learning_config import OptimizerType
+from InnerEye.ML.utils.split_dataset import DatasetSplits
+
+MODALITY = "mri"
+LABELS = ["hippocampus_L", "hippocampus_R"]
+
+class Hippocampus(SegmentationModelBase):
+    """
+    Hippocampus segmentation in MR scans.
+    """
```
Comment (fepegar) [inline InnerEye/ML/configs/segmentation/Hippocampus.py:20]: These three lines should be merged: `"""Hippocampus segmentation in MR scans."""`

File: InnerEye/ML/configs/segmentation/Hippocampus.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            use_model_parallel=True,
+        )
+        self.add_and_validate(kwargs)
+
+    def get_model_train_test_dataset_splits(self, dataset_df: pd.DataFrame) -> DatasetSplits:
+        return DatasetSplits.from_proportions(
+            dataset_df,
+            proportion_train=0.8,
+            proportion_val=0.1,
+            proportion_test=0.1,
```
Comment (fepegar) [inline InnerEye/ML/configs/segmentation/Hippocampus.py:64]: Are these the same proportions we've been using in our experiments?

**Review by fepegar — CHANGES_REQUESTED** [review]
I requested some minor changes. Can you please also remove the trailing whitespaces so the pre-commit check passes?

File: InnerEye/ML/configs/segmentation/Hippocampus.py (hunk 1 above)
Comment (annaschroder) [reply InnerEye/ML/configs/segmentation/Hippocampus.py:20]: Done, thanks!

File: InnerEye/ML/configs/segmentation/Hippocampus.py (hunk 2 above)
Comment (annaschroder) [reply InnerEye/ML/configs/segmentation/Hippocampus.py:64]: yes, these are the proportions we've been using

File: InnerEye/ML/configs/segmentation/Hippocampus.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+class Hippocampus(SegmentationModelBase):
+    """
+    Hippocampus segmentation in MR scans.
+    """
+
+    def __init__(self, **kwargs: Any) -> None:
+        fg_classes = LABELS
+        super().__init__(
+            should_validate=False,
+            azure_dataset_id="adni_hippocampus_L-R_split_minus_holdout",
```
[the remark this answers is not in the document]
Comment (annaschroder) [reply InnerEye/ML/configs/segmentation/Hippocampus.py:26]: That makes sense, thank you!

**Review by fepegar — APPROVED** [review]
LGTM. I assume the Azure pipelines errors are unrelated.

**Pull request merged**